timemachine/fe/rbfe.py

```diff
@@ -630,11 +648,22 @@ def run_vacuum(
         n_eq_steps=n_eq_steps,
         n_windows=n_windows,
         steps_per_frame=steps_per_frame,
+        image_traj=image_traj,
```

Comment:
Could probably force this to False all the time, as imaging vacuum frames doesn't do much

Reply:
still nice to have all legs have the same api